kiwi/builder/live.py

```diff
@@ -236,6 +237,15 @@ def create(self):
                 iso_header_offset, self.mbrid, self.isoname
             )
 
+        # include metadata for checkmedia tool
+        if self.checkiso is True and self.arch not in ['x86_64', 'ix86']:
+            log.warning(
+                'checkiso attribute is only supported for x86 architecture '
+                'family'
+            )
```

Comment:
well if the mediacheck is globally requested but can't be done when building for another architecture my opinion is that the author of the image description either has to create an arch specific profile in the XML description or use different image configurations. I'm not a friend of the implicit handling of arch specific differences. We have that in kiwi v7 and in obs to no end and it always confuses me when things are different but not made explicit in the description